mathics/builtin/numbers/calculus.py

```diff
@@ -1997,9 +2003,10 @@ def pre_makeboxes(self, x, x0, data, nmin, nmax, den, form, evaluation):
                         Expression(SymbolPower, variable, powers[i]),
                     )
             expansion.append(term)
-        expansion = ListExpression(
+        expansion = Expression(
+            SymbolList,
             Expression(SymbolPlus, *expansion),
-            Expression(SymbolPower, Expression(SymbolO, variable), powers[-1]),
+            Expression(SymbolPower, Expression("O", variable), powers[-1]),
```

Comment:
This seems like a regression, no?

Reply:
Sorry for that. I missed it in the rebase. I think now it is like in master